src/pybind/mgr/dashboard/frontend/src/locale/messages.zh-CN.xlf

```diff
@@ -3,75 +3,71 @@
     <body>
       <trans-unit id="319e0745bcbc132451569294fa2fa21bf10f555a" datatype="html">
         <source>Toggle navigation</source>
-        <target>切换导航</target>
+        <target>Toggle navigation</target>
```

Comment:
It seems to me like if all language files now contain the same strings (I haven't compared them with `diff`). I'm nearly convinced by now that something is wrong.